v2/charts/azure-service-operator/values.yaml

```diff
@@ -45,3 +45,8 @@ image:
   repository: mcr.microsoft.com/k8s/azureserviceoperator:v2.0.0-alpha.6
   kubeRBACProxy: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.1 
 
+# 'metrics' define settings for the metrics from controller.
+# 'address' field defines the metrics binding address on which metrics
+metrics:
+  enable: true
```

Comment:
Where does this flag end up getting passed?

Reply:
Basically this doesn't end up being a flag.. turning this to false omits the `metrics-addr` flag which results in a default `disable`